src/applications/gi/components/profile/EstimatedBenefits.jsx

```diff
@@ -14,7 +36,16 @@ const CalculatorResultRow = ({ id, label, value, header, bold, visible }) =>
         {header ? <h4>{label}:</h4> : <div>{label}:</div>}
       </div>
       <div className="small-6 columns vads-u-text-align--right">
-        {header ? <h5>{value}</h5> : <div>{value}</div>}
+        {header ? (
+          <div>
+            <h5>
+              {value}
+              {ScreenReaderSpan}
+            </h5>
+          </div>
+        ) : (
+          <div>{value}</div>
```

Comment:
should this also have the `{screenReaderSpan}`?

Reply:
It only has the span if it has a header in calculatorResultRow